eclipse-collections/src/main/java/org/eclipse/collections/impl/block/procedure/NonMutatingAggregationProcedure.java

```diff
@@ -47,4 +47,12 @@ public void value(T each)
         K key = this.groupBy.valueOf(each);
         this.map.updateValueWith(key, this.zeroValueFactory, this.nonMutatingAggregator, each);
     }
+
+    public void forEachWithOccurrences(T each, int occurrences)
```

Comment:
Hi @prathasirisha, I'm struggling on this name as well. On the one hand this name matches the parameters exactly. But I wonder if it would be better to name this `repeatForOccurrences`. @nikhilnanivadekar @motlin opinions?

Reply:
if @nikhilnanivadekar and @motlin are not in disagreement then let's go with `repeatForOccurrences`